sdks/go/pkg/beam/combine.go

```diff
@@ -23,22 +23,24 @@ import (
 
 // Combine inserts a global Combine transform into the pipeline. It
 // expects a PCollection<T> as input where T is a concrete type.
-func Combine(s Scope, combinefn interface{}, col PCollection) PCollection {
-	return Must(TryCombine(s, combinefn, col))
+// Combine supports only TypeDefinition options, not side inputs.
```

Comment:
I'd consider just saying TypeDefinition options, rather than clarifying the lack of support for SideInput. If we add additional Options in the future, it would be useful to not worry about updating the comment, unless we're also updating Combine anyway.

Here and for CombinePerKey.

Reply:
Thanks! I have revised the comments accordingly. PTAL.